lib/ExpensiMark.js

```diff
@@ -163,11 +163,19 @@ export default class ExpensiMark {
                 replacement: '<br />',
             },
             {
-                // when <pre> and <br /> occur together extra line is added so removing <br />
+                //eslint-disable-next-line no-useless-escape
```

Comment:
I think we don't want to force this eslint rule, because the original regex `/<\/pre>\s*<br\s*[/]?>/gi` was already valid, here is a test https://regexr.com/6k3dr

Reply:
should I not add the eslint rule ? Also, @marcochavezf please verify the regex here https://regex101.com/. I know this is strange that this regex is working fine and one site is saying its valid and other is saying its not. Although this doesn't seem correct to me. @rushatgabhane i did run npx eslint --fix before the last commit.

Reply:
@aneequeahmad ah that's because you're using the PHP version, please change it to ECMAScript (Javascript) version
<img width="644" alt="Screen Shot 2022-04-21 at 15 05 43" src="https://user-images.githubusercontent.com/6829422/164543701-e4716d7a-16b9-48eb-928f-2143284a4a63.png">

Reply:
TIL :)

@aneequeahmad sorry that I wasn't clear. I meant `npm run test`

Reply:
Yup, that can be confusing, I also thought that was fine until I saw the eslint errors too 😅